guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb

```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you rewrite the sentence "To begin with ... numeric values in your data".</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>